From 9ae1fb9c7e8bc8f6a66c2692a573c546ff56646a Mon Sep 17 00:00:00 2001
From: Jonas Dujava <jonas.dujava@gmail.com>
Date: Tue, 10 Oct 2023 17:04:57 +0200
Subject: [PATCH 1/2] docs: fix typo

---
 lua/notify/stages/util.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lua/notify/stages/util.lua b/lua/notify/stages/util.lua
index cc68974..d500a46 100644
--- a/lua/notify/stages/util.lua
+++ b/lua/notify/stages/util.lua
@@ -135,7 +135,7 @@ function M.available_slot(existing_wins, required_space, direction)
   return interval.min
 end
 
----Gets the next slow available for the given window while maintaining its position using the given list.
+---Gets the next slot available for the given window while maintaining its position using the given list.
 ---@param win number
 ---@param open_windows number[]
 ---@param direction string
-- 
2.42.0

